components/function-controller/internal/gitops/git.go

```diff
@@ -0,0 +1,21 @@
+package gitops
```

Comment:
I would name it `git` because it is only for communication with git repositories.